### ISSUE TITLE
Analysis: Understanding sequence output embeddings

### ISSUE THREAD

**Issue by PraljakReps** [PraljakReps/ProtWaveVAE_model#3]
Before applying the linear layer to convert model embeddings into logits, can we analyze and understand whether these embeddings lead to good representations for protein sequences? This will probably conducted with the unsupervised model to verify that ProtARDM can learn representations from unsupervised. In addition, FLIP and TAPE are popular benchmarking tasks, so understanding these representations and how they are interpreted from a diffusion modeling perspective is important for potentially using ProtARDM for regression. 

Goal for now: drop the final linear layer and layer normal. embed sequence represetnations and using low-dim visualization tools (e.g. tsne) to verify that the representations look promising.
